src/view/timer.js

```diff
@@ -51,22 +60,109 @@
     $(".timer__start-button").click(function(){
       const skillName = $(".timer__select-skill").val();
       const skill = LifeGamification.skillsCollection[skillName];
+      const typeName = $('.timer__task-type').val();
+      let info = {};
+      if(typeName === "countdown"){
+        info["countdown"] = $('#countdown').val();
+      } else if (typeName === "pomodoro"){
+        info["length"] = $('#pomodoro-length').val();
+        info["br"] = $('#pomodoro-break-length').val();
+        info["between"] = $('#pomodoro-between').val();
+        info["bigbr"] = $('#pomodoro-big-break-length').val();
+        info["number"] = $('#pomodoro-number').val();
+      }
+      const taskType = {"name": typeName, "info": info};
+
       if(!skill.timer.startTime){
-        LifeGamification.models.startWork(skill, "normal")
+        LifeGamification.models.startWork(skill, taskType)
           .then(LifeGamification.main.resetView);
       } else{
         console.log("Finish your current work first!");
       }
     });
+    $('.timer__task-type').change(function(){
+      if($('.timer__task_type').val() === "normal"){
+        $('.timer__pomodoro-options').html("");
+      }
+      else if($('.timer__task-type').val() === "countdown"){
+        $('.timer__pomodoro-options').html(`
+          <p>Countdown: <input id="countdown" type="number" value="60"> minutes</p>
+        `);
+      }
+      else if($('.timer__task-type').val() === "pomodoro"){
+        $('.timer__pomodoro-options').html(`
+          <p>Length of a pomodoro: <input id="pomodoro-length" type="number" value="25"> minutes</p>
+          <p>Length of a break: <input id="pomodoro-break-length" type="number" value="5"> minutes</p>
+          <p>Number of pomodoros between big breaks: <input id="pomodoro-between" type="number" value="4"></p>
+          <p>Length of a big break: <input id="pomodoro-big-break-length" type="number" value="30"> minutes</p>
+          <p>Number of pomodoros: <input id="pomodoro-number" type="number" value="8"></p>
+        `);
+      }
+    });
   }
 
   const handleTimer = function (skillsView) {
```

Comment:
This function is too long and do everything. It is unreadable because of it's length. I think you should add CountDownTimer class to models and then we will think what to do.